actuators/machine.go

```diff
@@ -81,14 +80,9 @@ func (m *Machine) Create(ctx context.Context, c *clusterv1.Cluster, machine *clu
 				m.Log.Error(err, "Error adding control plane")
 				return err
 			}
-			nodeUID, err := actions.GetNodeRefUID(c.GetName(), controlPlaneNode.Name())
-			if err != nil {
-				m.Log.Error(err, "Error getting node reference UID")
-				return err
-			}
-			providerID := providerID(controlPlaneNode.Name())
+			providerID := actions.ProviderID(controlPlaneNode.Name())
```

Comment:
We no longer have to set the machine ProviderID as the node ref controller will do that for us

Reply:
Hi @chuckha,

I thought you're **required** to set the machine's ProviderID in order for the NodeRef controller to match the machine with the node?

Reply:
you're right. I definitely meant that we don't have to set the machine's NodeRef. We still do have to set the provider ID

Reply:
Thank you for the clarification. For a moment I thought my understanding of this process was completely backwards :)